cmd/kops/cmds/create_cluster.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package cmds
```

Comment:
Consider the following layout instead.

https://github.com/kubernetes/kubernetes/blob/master/cmd/kubectl/kubectl.go#L29

Reply:
Generally agree but the kops project is so spread I doubt starting with this would help to clean this up. @mikesplain @justinsb what do you say?

Reply:
requesting feedback from @mikesplain @justinsb

Reply:
As this line refers to the package name and does not import anything this looks perfectly fine to me. 

All projects do it as proposed here, see: 
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubectl/kubectl.go#L17